### DISCUSSION TITLE
ESP32-Mini-1 support in Zephyr

### DISCUSSION THREAD

**Discussion by PragatiGarg-eaton** [zephyrproject-rtos/zephyr#81477] [General] [answered]
Hi All, 

I am currently exploring the capabilities of the Zephyr RTOS and am particularly interested in the ESP32-Mini-1 module.

Could anyone please confirm whether the ESP32-Mini-1 is supported by Zephyr? So far I couldn't find the board support added under \zephyr\boards\xtensa\ folder. If so, I would appreciate any insights on the specific features or limitations associated with using this module within the Zephyr environment.

Thanks in advance! 🙌

**Comment by iamadahn** [comment]
Hello,
If you are talking about this chip - https://www.espressif.com/sites/default/files/documentation/esp32-mini-1_datasheet_en.pdf
Then according to - https://www.espressif.com/en/products/socs it is in the same series with ESP32-D0WD, which is ESP32-WROOM, one of the most popular chips, so it has to be supported by Zephyr RTOS.

**Reply by PragatiGarg-eaton** [reply]
Hi @iamadahn @sylvioalves , I see some discrepancies here. The SoC for [ESP32-MINI-1](https://www.espressif.com/sites/default/files/documentation/esp32-mini-1_datasheet_en.pdf)' is the ESP32-U4WDH, not the ESP32-D0WD. Please let me know if I’ve overlooked anything.
![image](https://github.com/user-attachments/assets/0b6aa246-792f-4756-b4a8-926d3175be1c)

**Reply by iamadahn** [reply]
I was trying to say that both of this chips are in the same series, so they must be equally supported. In fact, the only major difference between them is that ESP32-U4 has 4MB of in-package flash, while ESP32-D0 is supposed to work with external flash

**Comment by sylvioalves** [comment, marked as answer]
> Hi All,
> 
> I am currently exploring the capabilities of the Zephyr RTOS and am particularly interested in the ESP32-Mini-1 module.
> 
> Could anyone please confirm whether the ESP32-Mini-1 is supported by Zephyr? So far I couldn't find the board support added under \zephyr\boards\xtensa\ folder. If so, I would appreciate any insights on the specific features or limitations associated with using this module within the Zephyr environment.
> 
> Thanks in advance! 🙌

Hi, it is supported, as you can see here:
https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/espressif/esp32/Kconfig.soc#L22

I also suggest you to use latest Zephyr release if possible so you get updated environment.
`zephyr\boards\xtensa\` path you mentioned is a bit old now as boards are place in `zephyr/boards/vendor`.

Sylvio
